Plugins/NativeEngine/CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ add_library(NativeEngine ${SOURCES})
 
 target_include_directories(NativeEngine PUBLIC "Include")
 
+if(WIN32 AND NOT WINDOWS_STORE)
+    set_property(TARGET NativeEngine PROPERTY UNITY_BUILD false)
```

Comment:
Is this temporary? Will this cause the size to be big again?

Reply:
BRN is UWP. WINDOW_STORE == true in that case.

Reply:
Ah, okay.

Reply:
Do we even need UNITY_BUILD overall if it's not UWP?

Reply:
I'm curious to see the impact on overall build time. But yes, otherwise, only usefull for UWP.